integrations/links_test.go

```diff
@@ -150,3 +150,26 @@ func TestLinksLogin(t *testing.T) {
 
 	testLinksAsUser("user2", t)
 }
+
+func testRepoLinksNoLogin(repoPath string, t *testing.T) {
+	prepareTestEnv(t)
+
+	var links = map[string]int{
+		"/": 200,
+	}
+	for link, expectedStatus := range links {
+		req := NewRequest(t, "GET", path.Join(repoPath, link))
+		MakeRequest(t, req, expectedStatus)
+	}
+}
+
+func TestRepoLinksNoLogin(t *testing.T) {
```

Comment:
Please add to the existing test in `integrations/repo_test.go`, instead of creating a duplicate.